debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -275,6 +275,16 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withDefault(DEFAULT_SNAPSHOT_LOCK_TIMEOUT_MILLIS)
             .withDescription("The maximum number of millis to wait for table locks at the beginning of a snapshot. If locks cannot be acquired in this " +
                     "time frame, the snapshot will be aborted. Defaults to 10 seconds");
+    public static final Field INCLUDE_SCHEMA_CHANGES = Field.create("include.schema.changes")
```

Comment:
@jpechane Looks like it's not added to any config group? Before doing so, probably a good idea to rebase this one, so you can do it the new way after the recent refactoring.